### ISSUE TITLE
commonjs vs ES6

### ISSUE THREAD

**Issue by EnriqueVidal** [dmtrKovalenko/reason-date-fns#105]
Hi, 

This is probably a dumb question but my project compiles to commonjs which node naturally accpets, however this is compiled to es6, I am using babel-node to run my code however this still gets picked up by node and I get complaints similar to: 

```
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /home/xyz/Code/nkdo/node_modules/reason-date-fns/src/DateFns.bs.js
require() of ES modules is not supported.
require() of /home/xyz/Code/nkdo/dashboard/node_modules/reason-date-fns/src/DateFns.bs.js from /home/xyz/Code/nkdo/dashboard/build/bundle.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename DateFns.bs.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /home/xyz/Code/nkdo/dashboard/node_modules/reason-date-fns/package.json.
```

My project is a mixed between reasonml and typescript using babel with the typescript preset to make them work, however even though bsb compiles and webpack can bundle the whole thing with no  problem whenever I try to run via `babel-node` (development) or `node` production I get these errors, I know about `"type": "module"` but that will break the commonjs (which is 99%) of the app.

My questions are, has anyone had this issue? and if so is there a work around this?

**Comment by dmtrKovalenko** [comment]
Thanks for raising this. I will investigate how it is solving in other reason definitions packages. Would be ideal to build both versions and publish them

**Comment by EnriqueVidal** [comment]
Hi @dmtrKovalenko I noticed you released both version, however I still have the same error being thown in the version fixed by #109:

```
is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename DateFns.bs.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /home/xyz/Code/nkdo/dashboard/node_modules/reason-date-fns/package.json.
```

I think it's hinting at the `"type": "module"` in your package.json for this library, the error puts it better than me, maybe if you want to keep that type we only need to change the extension?

**Comment by dmtrKovalenko** [comment]
Could you please check if this would work for you if you will remove `type: "module"` locally in node_modules?

**Comment by EnriqueVidal** [comment]
@dmtrKovalenko  that indeed fixes things and starts up my app:

```
yarn start
yarn run v1.22.4
$ npm-run-all --parallel dev:*
$ bsb -make-world -w
$ cross-env NODE_ENV=development BABEL_ENV=node NODE_PATH=./src nodemon -e .js,.jsx,.ts,.tsx --exec babel-node --extensions ".js,.ts,.jsx,.tsx" ./src/server
bsb: no work to do.
bsb: no work to do.
bsb: no work to do.
bsb: no work to do.
bsb: no work to do.
bsb: no work to do.
bsb: no work to do.
bsb: no work to do.
bsb: no work to do.
bsb: no work to do.
bsb: no work to do.
bsb: no work to do.
bsb: no work to do.
[nodemon] 2.0.4
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: js,jsx,ts,tsx
[nodemon] starting `babel-node --extensions .js,.ts,.jsx,.tsx ./src/server`
bsb: no work to do.
bsb: no work to do.
bsb: no work to do.
bsb: no work to do.
bsb: no work to do.
bsb: no work to do.
>>>> Start compiling 
bsb: no work to do.
>>>> Finish compiling 14 mseconds
Listening on port 8080. Open up http://localhost:8080/ in your browser.
webpack built ca409d0a781a04bf3828 in 3760ms
ℹ ｢wdm｣: Compiled successfully.
```

**Comment by dmtrKovalenko** [comment]
Released in 1.0.1
